src/pages/workspace/accounting/xero/export/XeroPreferredExporterSelectPage.tsx

```diff
@@ -0,0 +1,76 @@
+import isEmpty from 'lodash/isEmpty';
+import React, {useCallback, useMemo} from 'react';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import SelectionScreen from '@components/SelectionScreen';
+import * as Connections from '@libs/actions/connections';
+import {getAdminEmployees} from '@libs/PolicyUtils';
+import Navigation from '@navigation/Navigation';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type CardListItem = ListItem & {
+    value: string;
+};
+
+function XeroPreferredExporterSelectPage({policy}: WithPolicyConnectionsProps) {
+    const {export: exportConfiguration} = policy?.connections?.xero?.config ?? {};
+    const policyOwner = policy?.owner ?? '';
+    const exporters = getAdminEmployees(policy);
+
+    const policyID = policy?.id ?? '';
+    const data: CardListItem[] = useMemo(() => {
+        if (!isEmpty(policyOwner) && isEmpty(exporters)) {
+            return [
+                {
+                    value: policyOwner,
+                    text: policyOwner,
+                    keyForList: policyOwner,
+                    isSelected: true,
+                },
+            ];
+        }
+        return exporters?.reduce<CardListItem[]>((vendors, vendor) => {
```

Comment:
Can we replace this reduce with:
```
 return exporters
        ?.filter(vendor => vendor.email) // Filter out vendors without emails
        .map(vendor => ({
            value: vendor.email,
            text: vendor.email,
            keyForList: vendor.email,
            isSelected: exportConfiguration?.exporter === vendor.email,
        })) ?? []

```

Reply:
I'd do a `reduce` rather than combining `map` and `filter` 😄